tests/integration/cmd_pref_config_test.go

```diff
@@ -186,7 +184,7 @@ var _ = Describe("odo preference and config command tests", func() {
 					paramValue: "https://github.com/sclorg/nodejs-ex",
 				},
 			}
-			helper.CmdShouldPass("odo", "create", "--s2i", "nodejs", "--project", commonVar.Project)
+			helper.CmdShouldPass("odo", "create", "--s2i", "nodejs", "--project", commonVar.Project, "--git", "https://github.com/sclorg/nodejs-ex")
```

Comment:
I have noticed that you are using 3 different repos in tests. Is there a reason for that?
Could we use this one everywhere?

Reply:
I am not sure which one is latest, there is one in odo-devfiles repo too. Shouldn't we consider that?

Reply:
I would go with https://github.com/sclorg/nodejs-ex as these scenarios are testing --s2i so it would be good to use an example that was built for s2i.

Reply:
or https://github.com/openshift/nodejs-ex depending on which one is newer.

Reply:
both are same

Reply:
https://github.com/openshift/nodejs-ex points to https://github.com/sclorg/nodejs-ex

Reply:
now only two kinds of repos are being used - https://github.com/openshift/nodejs-ex and https://github.com/sclorg/nodejs-ex
But both are same

Reply:
IIRC we were always trying not to use external dependent repo in test, no? I am saying this because earlier we got into trouble due to this. So why are we using `--git` again 😕